lambda_functions/execute_ssh_command_js/serverless.yml

```diff
@@ -124,6 +124,46 @@ functions:
             product: ls8_nbar_scene
             path: '/g/data/rs0/scenes/nbar-scenes-tmp/ls8/2018/??/output/nbar/'
             trasharchived: no
+      - schedule:
+          rate: cron(10 10 ? * FRI *) # Run every Friday, at 08:10 pm Canberra time
+          enabled: true
+          input:
+            year: 2018
+            product: ls7_pq_scene
+            path: '/g/data/rs0/scenes/pq-scenes-tmp/ls7/2018/'
+            trasharchived: no
+      - schedule:
+          rate: cron(10 10 ? * FRI *) # Run every Friday, at 08:10 pm Canberra time
+          enabled: true
+          input:
+            year: 2018
+            product: ls7_level1_scene
```

Comment:
We don't need to index `level1` scenes.

Reply:
Removed `level1` scene processing from the configuration.